spec.html

```diff
@@ -27034,7 +27057,87 @@ <h1>_NativeError_.prototype.name</h1>
+        </ul>
+
+        <emu-clause id="sec-aggregate-error">
+          <h1>AggregateError ( _errors_, _message_ )</h1>
+          <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
+          <emu-alg>
+            1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
+            1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
+            1. If _message_ is not *undefined*, then
+              1. Let _msg_ be ? ToString(_message_).
+              1. Let _msgDesc_ be the PropertyDescriptor { [[Value]]: _msg_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
+              1. Perform ! DefinePropertyOrThrow(_O_, *"message"*, _msgDesc_).
+            1. Let _errorsList_ be ? IterableToList(_errors_).
+            1. Perform ! DefinePropertyOrThrow(_O_, `"errors"`, Property Descriptor { [[Configurable]]: *true*, [[Enumerable]]: *false*, [[Writable]]: *true*, [[Value]]: ! CreateArrayFromList(_errorsList_) }).
+            1. Return _O_.
+          </emu-alg>
+        </emu-clause>
+      </emu-clause>
+
+      <emu-clause id="sec-properties-of-the-aggregate-error-constructors">
+        <h1>Properties of the AggregateError Constructor</h1>
+        <p>The AggregateError constructor:</p>
+        <ul>
+          <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error%.</li>
+          <li>has the following properties:</li>
+        </ul>
+
+        <emu-clause id="sec-aggregate-error.prototype">
+          <h1>AggregateError.prototype</h1>
+          <p>The initial value of `AggregateError.prototype` is the intrinsic object %AggregateError.prototype%.</p>
+          <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+        </emu-clause>
+      </emu-clause>
+
+      <emu-clause id="sec-properties-of-the-aggregate-error-prototype-objects">
+        <h1>Properties of the AggregateError Prototype Object</h1>
+        <p>The AggregateError prototype object:</p>
+        <ul>
+          <li>is the intrinsic object <dfn>%AggregateError.prototype%</dfn>.</li>
+          <li>is an ordinary object.</li>
+          <li>is not an Error instance or an AggregateError instance and does not have an [[ErrorData]] internal slot.</li>
+          <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error.prototype%.</li>
+        </ul>
+
+        <emu-clause id="sec-aggregate-error.prototype.constructor">
+          <h1>AggregateError.prototype.constructor</h1>
+          <p>The initial value of `AggregateError.prototype.constructor` is the intrinsic object %AggregateError%.</p>
+          <p>This property has the attributes { [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.</p>
```

Comment:
No need to explicitly list the default attribute values: https://tc39.es/ecma262/#_ref_5389:~:text=Every%20other%20data%20property%20described%20in,%5B%5BConfigurable%5D%5D%3A%20true%20%7D%20unless%20otherwise%20specified.